docs/articles_en/documentation/openvino_legacy_features.rst

```diff
@@ -29,7 +29,7 @@ offering.
 |
 |   OpenVINO Development Tools used to be the OpenVINO package with tools for
     advanced operations on models, such as Model conversion API, Benchmark Tool,
-    Accuracy Checker, Annotation Converter, Post-Training Optimization Tool,
+    Accuracy Checker, Annotation Converter,
```

Comment:
this is a statement about historical state - no need to remove. Especially with the description following.